<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INLEN6239019998" failures="2" tests="10" name="RetryMultipleFailedCases" time="7.318" errors="0" timestamp="2023-01-19T12:15:00 IST">
  <testcase classname="testexecution.HomePageTest" name="@BeforeMethod setUp" time="7.25">
    <failure type="org.openqa.selenium.InvalidArgumentException" message="invalid argument
  (Session info: chrome=109.0.5414.75)
Build info: version: &amp;apos;4.4.0&amp;apos;, revision: &amp;apos;e5c75ed026a&amp;apos;
System info: host: &amp;apos;INLEN6239019998&amp;apos;, ip: &amp;apos;192.168.1.103&amp;apos;, os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.5&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [e4325f3972677efb9348b2c5fac16b32, get {url=}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 109.0.5414.75, chrome: {chromedriverVersion: 109.0.5414.74 (e7c5703604da..., userDataDir: C:\Program Files (x86)\scop...}, goog:chromeOptions: {debuggerAddress: localhost:55123}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:55123/devtoo..., se:cdpVersion: 109.0.5414.75, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: e4325f3972677efb9348b2c5fac16b32">
      <![CDATA[org.openqa.selenium.InvalidArgumentException: invalid argument
  (Session info: chrome=109.0.5414.75)
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'INLEN6239019998', ip: '192.168.1.103', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [e4325f3972677efb9348b2c5fac16b32, get {url=}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 109.0.5414.75, chrome: {chromedriverVersion: 109.0.5414.74 (e7c5703604da..., userDataDir: C:\Program Files (x86)\scop...}, goog:chromeOptions: {debuggerAddress: localhost:55123}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:55123/devtoo..., se:cdpVersion: 109.0.5414.75, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: e4325f3972677efb9348b2c5fac16b32
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:310)
at testexecution.HomePageTest.setUp(HomePageTest.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="testexecution.HomePageTest" name="validateHotelTitle" time="0.0">
    <skipped/>
  </testcase> <!-- validateHotelTitle -->
  <testcase classname="testexecution.DemoTest" name="test1" time="0.004"/>
  <testcase classname="testexecution.DemoTest" name="test2" time="0.002">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="testexecution.DemoTest" name="test2" time="0.001">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="testexecution.DemoTest" name="test2" time="0.0">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="testexecution.DemoTest" name="test2" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testexecution.DemoTest.test2(DemoTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="testexecution.DemoTest" name="test3" time="0.002">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="testexecution.DemoTest" name="test3" time="0.002">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="testexecution.DemoTest" name="test3" time="0.001">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="testexecution.DemoTest" name="test3" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testexecution.DemoTest.test3(DemoTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- RetryMultipleFailedCases -->
